styles.css

```diff
@@ -0,0 +1,140 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+* {
+  font-family: 'Crete Round', serif;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+li {
+  list-style-type: none;
+}
+
+a {
+  text-decoration: none;
+  color: white;
+}
+
+.burger {
+  display: none;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  justify-content: space-between;
+  background-color: #3e3c3c;
+  color: #fff;
+  height: 72px;
+}
+
+.brand {
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 20px;
+  margin-left: 7%;
+}
+
+.nav-list {
+  margin-right: 17%;
+  display: flex;
+  gap: 30%;
+}
+
+.first-page {
+  background-image: url(assests/bgimg/Illustration-header\ 1.png);
+  background-repeat: no-repeat;
+  background-size: contain;
+  height: 95vh;
+  background-color: #1c1a19;
+  background-position: right;
+  position: relative;
+}
+
+.contant {
+  color: #fff;
+  display: flex;
+  height: 95vh;
+}
+
+.social {
+  justify-content: center;
+  display: flex;
+  flex-direction: column;
+  gap: 25px;
+  margin-left: 2%;
+}
+
+.cont {
+  justify-content: center;
+  display: flex;
+  flex-direction: column;
+  margin-left: 10%;
+}
+
+h1 {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 72px;
+  line-height: 82px;
+  color: #f55800;
+}
+
+h3 {
+  font-family: 'Inter', sans-serif;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 44px;
+}
+
+p {
+  max-width: 566px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+}
```

Comment:
- [ ] The page responsibility should be applied to 375px screens at a minimum. I can see your titles' font sizes are greater than they should be. Your text overlaps the background picture and it makes it hard to read. Kindly fix it by applying the right positioning.
- [ ] Your social icons are not ordered as in the figma design. Kindly fix it.
yours:
|![image](https://user-images.githubusercontent.com/91486791/219036559-92a976b6-3da4-496f-9824-fa0a267ba5ad.png)
figma:
![image](https://user-images.githubusercontent.com/91486791/219036695-39793a0e-f036-454c-9597-ce5e8eecd596.png)|